### ISSUE TITLE
Twitch 'Go live' event

### ISSUE THREAD

**Issue by micheelengronne** [muesli/beehive#370]
I would like to have the 'Go live' event added to the Twitch bee.

If I have time I will try to do a commit for that.

**Comment by timkrief** [comment]
Yeah, that would be great to have :) Maybe it could be possible by getting a mail notification to a fake account? But it would add a lot of latency.
